### ISSUE TITLE
Feature: Left click to show Quick Actions popup

### ISSUE THREAD

**Issue by EddyErkel** [LAB02-Research/HASS.Agent#197]
Left click on the tray-icon to show Quick Actions popup (instead of right click > Show Quick Actions).
Left click again to hide the Quick Actions popup.

**Comment by LAB02-Admin** [comment]
Good idea, added to the list: [hassagent-153](https://lab02research.youtrack.cloud/issue/hassagent-153), thanks :)

**Comment by enchained** [comment]
@LAB02-Admin I think that both right, left and double left click should be fully customizable. I found it really confusing that it's not possible to set up left click webview with right click default app menu - and I thought this is the most obvious choice since most software act this way in the tray - left click launches a small (or big) UI while right click gives a menu with settings/quit/etc.

List of actions to choose for those clicks would be:

- Open main app window
- Open default app menu
- Open webview
- Open quick action selector/popup

I agree with the main request about making quick actions popup one click less, but an option to add selected quick actions to the default app menu can also achieve the same thing and will be less invasive vs popup. This is just an idea on how to customize it further. 

I thought about it all when I considered installing HASS.Agent on my family PCs, and I'm sure they will rarely use it and won't remember any hotkeys, but definitely will appreciate a fast, intuitive/obvious way to do something with only a couple of clicks.
